cfme/utils/template/gce.py

```diff
@@ -66,10 +66,21 @@ def create_template(self):
         return True
 
     def run(self):
-        self.download_image()
-        self.create_bucket()
-        self.upload_image()
-        self.create_template()
+        if not self.download_image():
+            return False
+
+        if 'miq' in self.template_name:
```

Comment:
Again I'd much rather unzip_image just skip when the file isn't a zip or tag.gz archive, and not have to explicitly check upstream/downstream via some substring. If the miq_version.TemplateName changes format and starts using `manageiq` instead of `miq`, then this breaks.